modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -9,40 +9,78 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 
-External vulnerability scans, via an Approved Scanning Vendor (ASV), as described in section 11.2.2 of the PCI standard may be performed at will and do not require prior approval. Penetration testing as described by section 11.3.1 of the PCI standard does require prior approval from MuleSoft.
 
-CloudHub Workers, by default, do not use static IPs and cannot be properly tested as their IPs may change. Please enable static IPs, as described in xref:deploying-to-cloudhub.adoc#static-ips[Static IPs].
+== Request a Security Assessment 
 
-To perform such testing, you must provide a written request 5 business days in advance to security@mulesoft.com along with:
+To request a Security Assessment:
 
-* Start and end time of testing.
-* Hosts to include in the test.
-* Source IP address or range.
-* Contact information, including a phone number and email, for the individual or team conducting the test.
-* You agree to only audit CloudHub workers, and not audit any other platform services which are common to all tenants.
-* You agree not to audit shared workers.
-* Send an email to security@mulesoft.com upon start and completion of tests.
-* Notify security@mulesoft.com of any vulnerabilities found immediately.
-* You may only audit once a quarter.
+. Contact your Salesforce Account team to request the Security Assessment Request form and Security Assessment Agreement.
+. Complete the request form and accept the agreement.
++
+For information about how to complete the request form, see the knowledge article https://help.salesforce.com/articleView?id=000322336&type=1&mode=1[Complete the Security Assessment submission form].
+. Send your completed Security Assessment form to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].
++
+Your account representative contacts you when your Security Assessment request is approved.
 
-After you submit your request, you receive a written authorization from the MuleSoft team, at which point you can proceed with testing according to your proposed plan.
+== Run Penetration and Vulnerability Tests  
+
+Because penetration testing could interfere with other tenants, MuleSoft allows penetration testing 
+on your workers but not other Anypoint Platform services.
+
+=== Prerequisites
+
+Before initiating penetration testing, enable static IPs for the app.
+
+By default, CloudHub workers do not use static IP addresses, so you can't test them because their IP addresses might change.
+For information about how to enable static IPs, see xref:deploying-to-cloudhub.adoc#static-ips-tab-settings[Static IPs Tab Settings].
+
+
+=== Steps 
+
+After your Security Assessment request is approved, follow these steps:
+
+. Run the penetration tests.
++
+The Security Assessment Agreement includes restrictions and requirements for testing.
+. Assign a security resource at your company to review and validate findings from the tests.
++
+[IMPORTANT]
+You must complete this step before following up with Salesforce. 
```

Comment:
The step of assigning a resource...or the missing step of having that person review and validate?
Hmm...let me add a word or two above....